src/index.html

```diff
@@ -1,12 +1,492 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
```

Comment:
The stylesheet link should have a .css file, not a .scss file. Change the link href from './styles/main.scss' to './styles/main.css'